<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PgImpParamPage</class>
 <widget class="QWizardPage" name="PgImpParamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Файл дампа:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="dumpPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="selfileButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../FmtLib/res/fmt.qrc">
             <normaloff>:/img/openfolderHS.png</normaloff>:/img/openfolderHS.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Адрес сервера:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Порт:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="serverImpEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="portImpBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>5432</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="selectServer">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../FmtLib/res/fmt.qrc">
             <normaloff>:/img/dpplg.dll_I0095_0409.ico</normaloff>:/img/dpplg.dll_I0095_0409.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Имя администратора:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adminEdit">
         <property name="text">
          <string>postgres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Владелец базы:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ownerEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Пользователь, куда заливать:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userDestEdit"/>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="jarButton">
         <property name="text">
          <string>Дополнительные jar</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/img/jar.png</normaloff>:/img/jar.png</iconset>
         </property>
         <property name="description">
          <string>PLXmlFunctions.jar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
  <include location="../FmtLib/res/fmt.qrc"/>
 </resources>
 <connections/>
</ui>
